compose/assets/scripts/ci-run-testsuite.sh

```diff
@@ -3,12 +3,11 @@ set -ex
 
 OIDC_AGENT_ALIAS=${OIDC_AGENT_ALIAS:-dev-wlcg}
 
-eval $(oidc-agent --no-autoload)
-oidc-add --pw-cmd='echo ${OIDC_AGENT_SECRET}' ${OIDC_AGENT_ALIAS}
+eval $(oidc-agent-service use)
+oidc-add --pw-env=OIDC_AGENT_SECRET ${OIDC_AGENT_ALIAS}
 export IAM_ACCESS_TOKEN=$(oidc-token -s openid ${OIDC_AGENT_ALIAS})
```

Comment:
Check if it's still used

Reply:
Yes, it is used here https://github.com/italiangrid/storm-webdav/blob/certs-on-the-fly/robot/common/credentials.robot#L9. So I'm leaving this line.